wp-config.php

```diff
@@ -1,6 +1,6 @@
 <?php
 define('DB_NAME', $_SERVER['RDS_DB_NAME']);
-define('DB_USER', $_SERVER['RDS_USERNAME']);
+define('DB_USER', $_SERVER['RDS_HOSTNAME'] . ':' . $_SERVER['RDS_PORT']);
 define('DB_PASSWORD', $_SERVER['RDS_PASSWORD']);
 define('DB_HOST', $_SERVER['RDS_HOSTNAME']);

```

Comment:
Port should be appended to DB_HOST

Reply:
Yes. Looks like I messed that up when copying my changes into this repo.

Reply:
Fixed in latest commit.